config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml

```diff
@@ -2,7 +2,7 @@ presubmits:
   kubernetes-sigs/kubebuilder:
   - name: pull-kubebuilder-test
     decorate: true
-    always_run: true
+    skip_if_only_changed: "^docs/|^netlify\\.toml|\\.(md|MD|png|pdf)$|^(OWNERS|OWNERS_ALIASES|LICENSE)$"
```

Comment:
@camilamacedo86 What is wrong with this approach? Could you please explain. Or should we use reusable workflows instead like in this case 
https://github.com/kubernetes-sigs/kubebuilder/blob/647be49c440b42501d9eda63919e4062a98a7bf9/.github/workflows/unit-tests.yml#L10.

Reply:
I do not think that you have here the kubebuilder branch. 
I would recommend we do the skip in the shell script used on the test instead.

/hold

Reply:
That seems right. We will need to check this one.